posthog/schema.py

```diff
@@ -1220,6 +1220,14 @@ class HogQLQueryResponse(BaseModel):
     types: Optional[List] = Field(default=None, description="Types of returned columns")
 
 
+class InsightActorsQueryBase(BaseModel):
+    model_config = ConfigDict(
+        extra="forbid",
+    )
+    includeRecordings: Optional[bool] = None
```

Comment:
I'm actually not sure if this is ever `False`. It wasn't when I worked on it on paths. So we might just optimize this away and always return them.

Reply:
Thanks for the hint, I'll double check this. I think for funnels this might be false for correlation queries or correlation actor queries. Might make sense to remove it from the schema though.